todo_app/app.py

```diff
@@ -8,4 +9,13 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template("index.html", items=items)
+
+
+@app.route('/add_items', methods=["POST"])
+def create_items():
+    new_item = request.form.get('todo')
+    add_item(new_item)
+    return redirect("/")
```

Comment:
It would be better to use `url_for("index")` here rather than `"/"`.

That allows you to target the function name, rather than the route part.

Reply:
https://flask.palletsprojects.com/en/2.1.x/api/#flask.url_for

Reply:
Noted, will change!

Again, I think I used this to begin with and then got an error, but I think this was thrown by something else. Will try to resolve so that it works. Thank you!